tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/cassandra/CassandraTableManager.java

```diff
@@ -216,12 +216,12 @@ protected TableName createImmutableTableName(TableHandle tableHandle)
         );
     }
 
-    private void executeQueryIgnoreError(String sql)
+    private void executeQueryIgnoreTypeError(String sql)
     {
         try {
             queryExecutor.executeQuery(sql);
         }
-        catch (IllegalStateException e) {
+        catch (CassandraQueryExecutor.TypeNotSupportedException e) {
```

Comment:
Can you check the type before running the query? I prefer using `if` than `try-catch`